Update README.md

Antipin-A-A opened the pull request:
консольное приложение №1

Antipin-A-A committed:
Update README.md

 консольное приложение №1

Antipin-A-A committed:
консольное приложение №1

Antipin-A-A committed:
Merge remote-tracking branch 'origin/dev' into dev

Antipin-A-A committed:
консольное приложение №1

Antipin-A-A commented:
Попытка №3

ArturNurtdinov commented on src/main/java/Main.java:
```diff
@@ -1,6 +1,82 @@
+                scanner.next();
+            }
+        }
+        System.out.println("Колличество человек после = " + person);
```
Можно не выводить два раза количество персон, либо в 17й строке оставить, либо здесь

ArturNurtdinov commented on src/main/java/Main.java:
```diff
@@ -1,6 +1,82 @@
+                System.out.println("некорректное значение для подсчёта, введите положительное число ");
+                scanner.next();
+            }
+        }
```
Считывание кол-ва человек можно в отдельную функцию выделить

ArturNurtdinov commented on src/main/java/Main.java:
```diff
@@ -1,6 +1,82 @@
+            } else {
+                System.out.println("Введите стоимость товара руб,коп");
+                if (scanner.hasNextDouble()) {
+                    double priceInt = scanner.nextDouble();
```
Тут имя переменной вводит в заблуждение, перменная `priceInt`, а её тип - `double`

Antipin-A-A committed:
исправил вывод склонения рубля
подсчет людей вывел в в отдельную функцию
исправил переменную  priceInt, на priceDouble
❌ Считать сумму, которую должен заплатить каждый поровну - этот метод был в  double priseFoPerson = sum / person;
строка 45 считает всю сумму , а в строке 56 расчет на 1 человека

Antipin-A-A commented:
исправил

Antipin-A-A committed:
person = 0;

Antipin-A-A commented:
private int person = 1;   стоит 0 , увидел 1 после отправления если это критичная ошибка
